README.md

```diff
@@ -28,7 +33,7 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
-<!-- Replace this comment with your answer -->
+a) c=7, the operators are + (addition) and = (allowing c to get the new value returned by the operation).  We work right to left so we perform 5 + 2 and then assign it to c.
```

Comment:
Almost. We don't work right to left, we work from highest precedence to lowest precedence. And it just so happens that `=` has very low precedence.